models/lavender_haze/main.py

```diff
@@ -14,13 +14,18 @@
 from execute_model_runs import execute_sweep_run, execute_single_run
 
 warnings.filterwarnings("ignore")
-
+try:
+    from common_utils.model_path import ModelPath
+    from common_utils.global_cache import GlobalCache
+    GlobalCache["current_model"] = ModelPath.get_model_name_from_path(Path(__file__))
+except Exception:
+    pass
 logger = setup_logging("run.log")
```

Comment:
This change will have to be made in all main.py files to support splitting logs by model. If not, lavender_haze could pollute the common_logs dir.